relay_test.go

```diff
@@ -2058,3 +2063,21 @@ func copyHeaders(m map[string]string) map[string]string {
 	}
 	return copied
 }
+
+func waitForInboundConnection(listeningCh, callingCh *Channel) bool {
```

Comment:
I see `testutils.WaitFor` - should we use that?

Reply:
good call. thanks!